analysis/02_new_model_input/02.2_assign_builing_damage_togrid.md

```diff
@@ -0,0 +1,147 @@
+# Assignment of building damage data to grids from municipalities
```

Comment:
Everything in this file ran quite quickly for me -- but I guess you did a lot of preprocessing in QGIS. What were the steps that took too long in Python?